query-engine/connectors/sql-query-connector/src/row.rs

```diff
@@ -16,6 +17,40 @@ pub struct SqlRow {
     pub values: Vec<PrismaValue>,
 }
 
+impl SqlRow {
+    pub fn as_aggregation_results(self, aggregators: &[Aggregator]) -> Vec<AggregationResult> {
```

Comment:
We take `self` by-value, so regarding the Rust guidelines, this should be called `into_aggregation_results`.